### PR TITLE
Enable BEP20 custom coin registration

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

**Comment by Milerius** [comment]
Hey @smk762 if you can test the coin addition on each platform (qrc20, erc20 and bep20) let us know if you face any issue

File: atomic_defi_design/qml/Settings/AddCustomCoinModal.qml [hunk 1]
```diff
@@ -63,23 +64,53 @@ BasicModal {
...
+            prefix: "0x"
+            image: "qrc"
+            url: "https://explorer.qtum.org/tokens/search"
+            name: 'QTUM Insight'
+            coinType: CoinType.QRC20
+        }
+        ListElement {
+            text: "BEP-20"
+            prefix: ""
+            url: "https://bscscan.com/"
```
Comment (smk762) [inline atomic_defi_design/qml/Settings/AddCustomCoinModal.qml:105]: https://bscscan.com/tokens would save users a click to find the tokens list.

**Review by smk762 — CHANGES_REQUESTED** [review]
Functional on all OS.
Small UX change request - use https://bscscan.com/tokens for the BEP link so it leads to the token list.

File: atomic_defi_design/qml/Settings/AddCustomCoinModal.qml (hunk 1 above)
Comment (Milerius) [reply atomic_defi_design/qml/Settings/AddCustomCoinModal.qml]: solved in https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/907/commits/d9184d7d9d64f0cbe60b22895745df6803c787f5

**Review by smk762 — APPROVED** [review]

**Pull request merged**